enterprise/settings/test.py

```diff
@@ -293,3 +293,8 @@ def root(*args):
         }
     ],
 }
+
+
+INTEGRATED_CHANNELS_API_CHUNK_TRANSMISSION_LIMIT = {
+    'SAP': 1,
```

Comment:
Do we expect it to ever become more than 1?

Reply:
I do not see it becoming more than 1.